core/logging_api.php

```diff
@@ -125,11 +126,18 @@ function log_event( $p_level, $p_msg ) {
 
 	$t_php_event = $t_now . ' ' . $t_level . ' ' . $t_msg;
 
+	if( isset( $t_modifiers ) && !( is_writable ( $t_modifiers ) ) && $t_call_once ){
```

Comment:
Why isn't check just before the error_log method is called in the switch statement?